src/DeploymentProvider/DeploymentHelper.ts

```diff
@@ -422,4 +418,55 @@ export class DeploymentHelper {
         const data = fs.readFileSync(file);
         return JSON.parse(data.toString());
     }
+
+    private static async printLatestAppInstanceLog(client: asa.AppPlatformManagementClient, params: ActionParameters) {
+        const logStream = await this.logStreamConstructor(client, params);
+        const deploymentResource = await client.deployments.get(params.resourceGroupName, params.serviceName, params.appName, params.deploymentName);
+        const instances = deploymentResource.properties.instances;
+        let startTime = instances[0].startTime;
+        let instanceName = instances[0].name;
+
+        // print the newly created instance log
+        for (const tempInstance of instances) {
+            if (tempInstance.startTime > startTime) {
+                startTime = tempInstance.startTime;
+                instanceName = tempInstance.name;
+            }
+        }
+        let streamingUrl = `https://${logStream["baseUrl"]}/api/logstream/apps/${params.appName}/instances/${instanceName}`;
+        const logParams: any = {};
+        logParams['tailLines'] = 500;
+        logParams['limitBytes'] = 1024 * 1024;
+        logParams['sinceSeconds'] = 300;
+        logParams['follow'] = true;
+        const credentials = Buffer.from(`primary:${logStream["primaryKey"]}`).toString('base64');
+        const auth = { "Authorization" : `Basic ${credentials}` };
+        var url = new URL(streamingUrl)
+        url.search = new URLSearchParams(logParams).toString();
+        fetch(url, {method: 'GET', headers : auth })
+            .then(response => {
+            if (response.ok) {
```

Comment:
nit: it seems the indentation here is broken.

Reply:
Fixed